code/modules/clothing/head/misc_special.dm

```diff
@@ -218,3 +218,10 @@
 	hair_style = pick(GLOB.hair_styles_list - "Bald") //Don't want invisible wig
 	hair_color = "#[random_short_color()]"
 	. = ..()
+
+/obj/item/clothing/head/foilhat
+	name = "tinfoil hat"
+	desc = "Thought control rays, psychotronic scanning. Don't mind that, I'm protected cause I made this hat."
+	icon_state = "foilhat"
+	item_state = "foilhat"
+	armor = list(melee = 0, bullet = 0, laser = -5,energy = 0, bomb = 0, bio = 0, rad = -5, fire = 0, acid = 0)
```

Comment:
qutoes ples

Reply:
this comment confuses me

Reply:
drinking at 9AM is not ok!

Reply:
if you're asking about the quote it came from weird al's song 'foil' and if you're saying i shouldn't have a quote there then you shouldn't be part of the conspiracy

Reply:
He's talking about changing `melee=` to `"melee"=`, but you can ignore it since it should be done wholesale and this is the current standard for armor lists.

Reply:
Just do it here so I don't have to later, please